index.bs

```diff
@@ -435,13 +435,16 @@ has defined a number of properties for itself:
 *   <dfn for="user agent" export>brand</dfn> - The [=user agent=]'s commercial name (e.g.,
       "cURL", "Edge", "The World's Best Web Browser"), which MUST be shorter than 32 [=ASCII alpha=]
       characters.
-*   <dfn for="user agent" export>significant version</dfn> - The marketing version which includes
-      distinguishable web-exposed features (e.g., "72", "3", or "12.1"), corresponding to the
-      [=user agent=], or any of the brands in its [=brands=] list (e.g., rendering engine or
-      any other [=equivalence classes=] full version).
+*   <dfn for="user agent" export>form-factor</dfn> - The form-factor of a device, historically
+      represted as a &lt;[=deviceCompat=]&gt; token in the User-Agent string (e.g., "Tablet", "VR",
```

Comment:
https://compat.spec.whatwg.org/#devicecompat was just exported today, should give it a day or so for other tooling to catch up.